### DISCUSSION TITLE
Is it possible to exclude a certain blade file from being cached?

### DISCUSSION THREAD

**Discussion by deniztezcan** [spatie/laravel-responsecache#371] [Q&A] [answered]
Hi there,

Is it possible to exclude a certain blade file from being cached? Similarly to how the CSRF token is always inserted(/replaced) before sending the new (cached) response.

Thanks in advance,

**Comment by freekmurze** [comment, marked as answer]
You can use a `CacheProfile` mentioned in the read to help determine what can should be cached. This is the finest level the package has to offer.

**Reply by deniztezcan** [reply]
Thanks @freekmurze
